src/ExportConverters.mss

```diff
@@ -700,25 +700,6 @@ function ConvertPositionWithDurationToTimestamp (bobj) {
     return measureDuration & 'm+' & position;
 } //$end
 
-function ConvertTupletStyle (tupletStyle) {
```

Comment:
This was removed, but I can't see where it was replaced?

Reply:
This is just removal of unused code in commit 2430053674d41af1c482834d8602d4e87fea183f.